pydis_site/static/css/base/base.css

```diff
@@ -0,0 +1,40 @@
+html {
+    overflow:auto;
+}
+
+body.site {
+  display: flex;
+  min-height: 100vh;
+  flex-direction: column;
+}
+
+main.site-content {
+  flex: 1;
+}
+
+div.card.has-equal-height {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+}
+
+#django-logo {
+  padding-bottom: 2px;
+  background: url(https://static.djangoproject.com/img/logos/django-logo-negative.png) no-repeat center;
+  filter: grayscale(1) invert(0.02);
+  background-size: 52px 25.5px;
+  background-position: -1px -2px;
+  color: #00000000;
+}
+
+#bulma-logo {
+  padding-left: 18px;
+  height: 20px;
+  background: url(https://bulma.io/images/bulma-logo-white.png) no-repeat center;
```

Comment:
not sure if the django and bulma websites like when we're using them for static hosting like this, can we copy them here or is there copy issues stuff with that?

Reply:
I seriously doubt they'd have issue with us using their image to give credit to them. Also, css can't use DTL, so it's tricky to fetch something from the static folder, which means we'd have to host them somewhere and refer to that, or do something hacky like build the css with DTL templates..

So I think this is fine for now.